### PR TITLE
executer, Parse test outputs

### PR DESCRIPTION
**Pull request by RamLavi**
This PR implements the functions that parse the statistics recieved from the DPDK applicatin and trex application, in order to pass them to the results struct in a future PR

### CODE REVIEW

**Comment by RamLavi** [comment]
```
zeus09:kubevirt-dpdk-checkup $ TEST_NAMESPACE=ralavi-ns TEST_IMAGE=quay.io/ramlavi/kubevirt-dpdk-checkup:latest make test/e2e
docker run --rm \
           --volume /root/go/src/github.com/kiagnose/kubevirt-dpdk-checkup:/root/go/src/github.com/kiagnose/kubevirt-dpdk-checkup:Z \
           --volume /root/.kube:/root/.kube:Z \
           --workdir /root/go/src/github.com/kiagnose/kubevirt-dpdk-checkup \
           -e KUBECONFIG=/root/.kube/config \
           -e TEST_IMAGE=quay.io/ramlavi/kubevirt-dpdk-checkup:latest \
           -e TEST_NAMESPACE=ralavi-ns \
           -e NETWORK_ATTACHMENT_DEFINITION_NAME= \
           -e RUNTIME_CLASS_NAME= \
           -e TRAFFIC_GEN_IMAGE_URL= \
           docker.io/library/golang:1.19 go test ./tests/... -test.v -test.timeout=1h -ginkgo.v -ginkgo.timeout=1h
=== RUN   TestKubevirtDpdkCheckup
Running Suite: KubevirtDpdkCheckup Suite - /root/go/src/github.com/kiagnose/kubevirt-dpdk-checkup/tests
=======================================================================================================
Random Seed: 1675674939

Will run 1 of 1 specs
------------------------------
[BeforeSuite] 
/root/go/src/github.com/kiagnose/kubevirt-dpdk-checkup/tests/test_suite_test.go:61
[BeforeSuite] PASSED [0.004 seconds]
------------------------------
Execute the checkup Job should complete successfully
/root/go/src/github.com/kiagnose/kubevirt-dpdk-checkup/tests/checkup_test.go:81
• [150.766 seconds]
------------------------------

Ran 1 of 1 Specs in 150.771 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 0 Skipped
--- PASS: TestKubevirtDpdkCheckup (150.77s)
PASS
ok  	github.com/kiagnose/kubevirt-dpdk-checkup/tests	150.801s

```

File: pkg/internal/checkup/executor/trex-console.go [hunk 1]
```diff
@@ -23,32 +24,56 @@ func NewTrexConsole(client podExecuteClient, namespace, name, containerName stri
...
+		packetPerSecondMillion, port, testDurationMinutes), false)
 }
 
 func (t trexConsole) StopTraffic(ctx context.Context) (string, error) {
-	return t.runCommand(ctx, "stop -a")
+	return t.runCommand(ctx, "stop -a", false)
 }
 
-func (t trexConsole) runCommand(ctx context.Context, command string) (string, error) {
+func (t trexConsole) runCommand(ctx context.Context, command string, returnJSONString bool) (string, error) {
```
Comment (RamLavi) [inline pkg/internal/checkup/executor/trex-console.go:98]: #2DO RAM 
split to 2 functions

File: pkg/internal/checkup/executor/trex-console.go [hunk 2]
```diff
@@ -52,6 +56,31 @@ func (t trexConsole) GetGlobalStats(ctx context.Context) (globalStats, error) {
 	return gs, nil
 }
 
+func (t trexConsole) MonitorDropRates(ctx context.Context, duration time.Duration) (float64, error) {
+	const interval = 10 * time.Second
+	log.Printf("Monitoring traffic generator side drop rates every %ss during the test duration...", interval)
+	maxDropRateBps := float64(0)
+
+	ctxWithNewDeadline, cancel := context.WithDeadline(ctx, time.Now().Add(duration))
```
Comment (RamLavi) [inline pkg/internal/checkup/executor/trex-console.go:64]: 2DO RAM
change to context.WithTimeout

**Review by orelmisan — APPROVED** [review]
Minimal review was performed in favor of pushing forward.

**Pull request merged**